src/commands/moderation/purge.ts

```diff
@@ -17,14 +17,15 @@ export default class Purge {
     this.data = new SlashCommandSubcommandBuilder()
       .setName("purge")
       .setDescription("Purges messages.")
-      .addNumberOption((number) =>
-        number.setName("amount").setDescription("The amount of messages that you want to purge.").setRequired(true)
+      .addNumberOption(number => number
+        .setName("amount")
+        .setDescription("The amount of messages that you want to purge.")
+        .setRequired(true)
       )
-      .addChannelOption((channel) =>
-        channel
-          .setName("channel")
-          .setDescription("The channel that you want to purge (if not the current channel)")
-          .addChannelTypes(ChannelType.GuildText, ChannelType.PublicThread, ChannelType.PrivateThread)
+      .addChannelOption(channel => channel
+        .setName("channel")
+        .setDescription("The channel that you want to purge (if not the current channel)")
```

Comment:
```suggestion
        .setDescription("The channel that you want to purge. Leave empty if you want to purge the current channel.")
```
This is more clear in my opinion

Reply:
next commit has it